### PR TITLE
Endpoints naming

### PR DESCRIPTION
**Pull request by gchatz22**

### CODE REVIEW

**Comment by gchatz22** [comment]
@jpbogle could you review the paths naming on this?

File: api/serverless.yml [hunk 1]
```diff
@@ -187,21 +187,35 @@ functions:
...
+      - http:
+          path: /tickets/create
+          method: post
+          cors: true
+  ticket-update:
     handler: events/ticketsManager/handler.manage
     events:
       - http:
-          path: /events/createOrUpdateTickets
+          path: /tickets/{ticketId}/update
```
Comment (jpbogle) [inline api/serverless.yml:215]: This should just be

`PUT /tickets/{ticketId}` 

PUT being a replace on that object

File: api/serverless.yml [hunk 2]
```diff
@@ -187,21 +187,35 @@ functions:
...
+    events:
+      - http:
+          path: /events/{eventId}/update
+          method: put
+          cors: true
+  ticket-create:
+    handler: events/ticketsManager/handler.manage
+    events:
+      - http:
+          path: /tickets/create
```
Comment (jpbogle) [inline api/serverless.yml:208]: can just be POST /tickets

File: api/serverless.yml [hunk 3]
```diff
@@ -187,21 +187,35 @@ functions:
...
-          path: /events/createOrUpdateEvent
+          path: /events/create
           method: post
           cors: true
-  events-tickets-manager:
+  event-update:
+    handler: events/eventManager/handler.update
+    events:
+      - http:
+          path: /events/{eventId}/update
```
Comment (jpbogle) [inline api/serverless.yml:201]: PUT /events/{eventId}

File: api/serverless.yml [hunk 4]
```diff
@@ -187,21 +187,35 @@ functions:
...
 
   ################################ Events ################################
-  events-event-manager:
-    handler: events/eventManager/handler.manage
+  event-create:
+    handler: events/eventManager/handler.create
     events:
       - http:
-          path: /events/createOrUpdateEvent
+          path: /events/create
```
Comment (jpbogle) [inline api/serverless.yml:194]: can just be POST /events

**Pull request merged**